### ISSUE TITLE
Enable learning rate scheduling

### ISSUE THREAD

**Issue by nrweir** [CosmiQ/cw-nets#52]
Enable learning rate scheduling for:
- [ ] tensorflow/keras
- [ ] pytorch
- [ ] darknet (? @avanetten)

**Comment by nrweir** [comment]
Issue moved to [CosmiQ/solaris #27](https://github.com/CosmiQ/solaris/issues/27) via [**ZenHub**](https://www.zenhub.com/)
